doc/manuals/admin/database_model.md

```diff
@@ -288,6 +291,13 @@ Fields:
     (meaning JSON notifications in NGSIv1 format), **normalized**, **keyValues** and **values** (the last three used in NGSIv2 format).
 -   **status**: either `active` (for active subscriptions) or `inactive (for inactive subscriptions).
 -   **description** (optional field): a free text string describing the subscription. Maximum length is 1024.
+-   **custom**: a boolean field to specify if this subscription uses customized notifications (a functionality in the NGSIv2 API).
+    If this field exist and its value is "true" then customized notifications are used and the `headers`, `qs`, `method` and
+    `payload` field are taken into account.
+-   **headers**: optional field to store the HTTP headers keymap for notification customization functionality in NGSIv2.
+-   **qs**: optional field to store the query parameters keymap for notification customization functionality in NGSIv2.
+-   **method**: optional field to store the HTTP method for notification customization functionality in NGSIv2.
+-   **payload**: optional field to store the payload for notification customization functionality in NGSIv2.
 
```

Comment:
I prefer to keep this file very focused in data model... the functionality related with the data model is described elsewhere. Thus, the reference to "customization fucntionality in NGSIv2" should be enough for the readers (supposed to be an "average reader" if he/she has ended reading this file :) as a pointer to the complete documentation.

NTC